### ISSUE TITLE
rat deleteColumn?

### ISSUE THREAD

**Issue by juan-guerschman** [ubarsc/rios#121]
can you delete a column using rios.rat?

**Comment by gillins** [comment]
No, sorry is a limitation of GDAL. I looked at implementing this just for KEA but found it would be quite hard with the way the RAT is stored in a KEA file.
Best way is to copy over just the columns you want into a new file with ratapplier. Or just leave the column there if it is not causing any harm.
